### DISCUSSION TITLE
#define IWIDTH  240 SIZE

### DISCUSSION THREAD

**Discussion by fungreenfox** [Bodmer/TFT_eSPI#1503] [Q&A - Sprites]
Sprite_scroll_8bit.ino

Found, that #define IWIDTH  240 can max be 2500. How to define a bigger amount, to make a longer text ?
Script will be used in a radio with RDS recieve option, and it is not known, how much longer the text string will be.

**Comment by Bodmer** [comment]
The processors memory is the limit.

Instead of creating a large sprite, create one that is the display width, then use a loop:
```
Clear sprite
Draw text in sprite at x,y
Draw sprite to screen
Decrement x
Repeat
```

**Comment by Bodmer** [comment]
Moving to discussions as this is not a library issue. Post back if you need more help.
